src/App.tsx

```diff
@@ -12,14 +12,17 @@ import { RunAllButton } from './components/RunAllButton/RunAllButton';
 import { DropdownSelect } from './components/DropdownSelect/DropdownSelect'
 
 const URL = "https://kroger-description-api-0b391e779fb3.herokuapp.com/"
+const wordLists = {
+    bannedWords: ['practical', 'straightforward', 'free form', 'wholesome', 'all', 'only', 'pure', 'real', 'healthy', 'boost', 'well-balanced', 'hand-crafted', 'all natural', 'natural', 'minimally precessed', 'processed', 'simple', 'protein packed', 'full of protein', 'muscle-building', 'freshness', 'fresh', 'source of energy', 'trace', 'clean', 'unadulterated', 'nutritious', 'healthiness', 'whisper', 'rich source', 'unparalleled', 'perfect', 'authentically', 'robust', 'hint', 'pinch', '100%'],
+    factCheckWords: ['protein', 'energy', 'omega 3 fatty acids', 'daily value', 'per serving', 'antioxidant', 'low fat', 'lowfat', 'gluten free', 'low sodium', 'low cholesterol', 'low saturated fat', 'good source', 'excellent source', 'whole wheat', 'light', 'reduced', 'added', 'added', 'extra', 'plus', 'fortified', 'enriched', 'more', 'less', 'high', 'rich in', 'contains', 'provides', 'lean', 'extra lean', 'high potency', 'modified', '\\bno\\b', 'free', 'zero', 'amount', 'keto', 'low carb', 'antibiotics', 'hormones', 'growth hormones', 'no sugar added', 'msg', 'cage free', 'made with' , 'fresh']
+}
```

Comment:
I'm sure there will be many tweaks needed to these words through no fault of yours. Let's change `light` to `\\blight\\` so it doesn't ping `delight` which is an okay word.